packages/labs/src/components/select/select.tsx

```diff
@@ -46,6 +46,18 @@ export interface ISelectProps<T> extends IListItemsProps<T> {
     itemRenderer: (itemProps: ISelectItemRendererProps<T>) => JSX.Element;
 
     /**
+     * Current query value, for controlled usage.
+     * Providing this prop will put the component in controlled mode.
```

Comment:
not the entire component, just this one feature... "Providing this prop will control the query input."